sysconfigs/daint/compilers.yaml

```diff
@@ -1,4 +1,21 @@
 compilers:
+- compiler:
```

Comment:
is it on purpose that there are also changes on the daint confg - the PR for daint (https://github.com/MeteoSwiss-APN/spack-mch/pull/203) was merged already, should there not only be dom modifications now ?

Reply:
I pull from master was needed to get rid of the daint changes.